agent/containers/images/Makefile

```diff
@@ -195,12 +218,12 @@ centos-8-base.Dockerfile: Dockerfile.base.j2 epel-8-pbench.repo
 centos-7-base.Dockerfile: Dockerfile.base.j2 epel-7-pbench.repo
 	jinja2 Dockerfile.base.j2 -D pbench_repo_file=epel-7-pbench.repo -D pkgmgr=yum -D distro_image=centos:7 -D distro_image_name="CentOS 7" -o $@
 
+fedora-33-base.Dockerfile: Dockerfile.base.j2 fedora-33-pbench.repo
```

Comment:
Can you split up this line, its kind of hard to read.

Reply:
Do you mean line 219?  Do you have any suggestions?